pkg/resource/deploy/plan_executor.go

```diff
@@ -36,9 +36,6 @@ type planExecutor struct {
 	stepExec *stepExecutor  // step executor owned by this plan
 }
 
-// Utility for convenient logging.
-var log = logging.V(4)
```

Comment:
how come?

Reply:
Turns out this doesn't work anymore - `logging.V(4)` gets the logging level at init, and we don't use the normal `glog` initialization mechanism (i.e. flags) anymore and set the log level after module constructors have run.